src/content/sessions/2023-09-01.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Rider: CE custom operations completion list"
+preview: ""
+isDraft: true
+date: 2023-09-01T13:00:00.000Z
+slug: "2023/09/01"
+champion: Jindřich Ivánek
+zoomLink: ""
+zoomPasscode: ""
+issueLink: "https://youtrack.jetbrains.com/issue/RIDER-15832/Put-custom-operations-at-the-top-of-completion-list-if-its-invoked-on-top-level-inside-computation-expression"
+company: "Ciklum"
+youtubeId: ""
+---
+
+## Topic
+
+Currently it's next to impossible to see what custom operations are available:
+
+<img alt="Currently it's next to impossible to see what custom operations are available" class="img-fluid" src="/images/sessions/ceCustomOperationCompletion.png"/>
+
+The goal of the session is to improve the situation by putting custom operations at the top of the completion list if it's invoked inside computation expression.
+
+## Champions
+
+- [Jindřich Ivánek](https://twitter.com/jindraivanek)
```

Comment:
Please change back to `https://github.com/jindraivanek`. I added link to socials and blog there. Thanks.

Reply:
Updated :)